$NetBSD: patch-SConstruct,v 1.1 2019/05/01 20:22:02 gdt Exp $

Tell scons to pass the wrapper config variable to children, so the
wrapped compiler works.

Not yet addressed upstream.

--- SConstruct.orig	2018-10-03 00:05:28.000000000 +0000
+++ SConstruct
@@ -280,6 +280,9 @@ import_env = (
     'STAGING_DIR',     # Required by the OpenWRT and CeroWrt builds.
     'STAGING_PREFIX',  # Required by the OpenWRT and CeroWrt builds.
     'WRITE_PAD',       # So we can test WRITE_PAD values on the fly.
+    # pkgsrc
+    'CWRAPPERS_CONFIG_DIR',
+    'CCACHE_DIR',
 )
 
 envs = {}
